domain/relation/state/relation.go

```diff
@@ -66,10 +69,166 @@ func (st *State) GetAllRelationDetails(ctx context.Context) ([]relation.Relation
 	return relationsDetails, errors.Capture(err)
 }
 
-// GetRelationID returns the relation ID for the given relation UUID.
+// GetPrincipalApplicationID return the principal application's ID given a
+// subordinate application's ID.
 //
 // The following error types can be expected to be returned:
-//   - [relationerrors.RelationNotFound] is returned if the relation UUID
+//   - [relationerrors.ApplicationNotSubordinate] is returned if the application
+//     RelationUUID is not found as a subordinate in the application_subordinate link table.
+//   - [relationerrors.ApplicationNotFound] is returned if the application
+//     RelationUUID is not found.
```

Comment:
Looks like the same renaming hit this too.

Reply:
rebase gone bad.